build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+buildscript {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10")
+        classpath("com.android.tools.build:gradle:7.2.2")
+        classpath("com.google.gms:google-services:4.3.13")
```

Comment:
same, versioning

Reply:
Replied here: https://github.com/suho/kmm-ic/pull/80#discussion_r964314582

I will create a chore to add `buildSrc`